### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Fireworkstars46** [AdvancedXRay/XRay-Mod#195] [General]
Can you make a 1.6.4 version of this mod?

**Comment by MichaelHillcox** [comment]
> Can you make a 1.6.4 version of this mod?

No, I have no plans to do that. Not only would it be a complex task, it's not worth my time for the limited player base for such an old version of the game.

**Comment by appletheIIII** [comment]
how do you use the mod IDK how
